### PR TITLE
Pv2 2789 timeout jobs running longer then average

### PR DESCRIPTION
**Pull request by AkshayBanifatsyevichChugh**

### CODE REVIEW

File: src/SFA.DAS.Payments.Monitoring.AcceptanceTests/App.config [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <add key="EndpointName" value="sfa-das-payments-monitoring-jobs-acceptancetests" />
     <add key="JobsServiceEndpointName" value="sfa-das-payments-monitoring-jobs" />
     <add key="Environment" value="DEVELOPMENT" />
-    <add key="TimeToWait" value="00:01:30" />
+    <add key="TimeToWait" value="00:02:30" />
```
Comment (AkshayBanifatsyevichChugh) [inline src/SFA.DAS.Payments.Monitoring.AcceptanceTests/App.config:17]: Was this change intended?

Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.Monitoring.AcceptanceTests/App.config:17]: this is was required because one of the new test requires job to time out and the current job timeout is 2 min so I needed to update this variable so that we wait longer then the job timeout

File: src/SFA.DAS.Payments.Monitoring.Jobs.Application/JobProcessing/JobStatusService.cs [hunk 2]
```diff
@@ -42,21 +42,26 @@ protected virtual Task<bool> CheckSavedJobStatus(JobModel job, CancellationToken
...
+
             var status = JobStatus.TimedOut;
             if (job.DcJobSucceeded.HasValue)
                 status = job.DcJobSucceeded.Value ? JobStatus.CompletedWithErrors : JobStatus.DcTasksFailed;
-                
-            return await CompleteJob(job, status, timedOutTime, cancellationToken)
-                .ConfigureAwait(false);
+
+            if (status != JobStatus.TimedOut) 
+                Logger.LogWarning($"Job {job.DcJobId} has timed out. but because DcJobSucceeded is {job.DcJobSucceeded}, CompleteJob Job as {status}");
```
[the remark this answers is not in the document]
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.Monitoring.Jobs.Application/JobProcessing/JobStatusService.cs:56]: may be the language is not quite clear but the log entry is trying to say that the job has been terminated due to timeout but because DcJobSucceeded is true or false the actual status we are going to save to DB is either CompletedWithErrors or DcTasksFailed

**Review by hardik-desai-kcl — APPROVED** [review]

**Review by hardik-desai-kcl — DISMISSED** [review]

**Comment by AkshayBanifatsyevichChugh** [comment]
reviewed

**Review by hardik-desai-kcl — DISMISSED** [review]

**Review by hardik-desai-kcl — DISMISSED** [review]

**Comment by AkshayBanifatsyevichChugh** [comment]
reviewed

**Review by hardik-desai-kcl — APPROVED** [review]

**Comment by AkshayBanifatsyevichChugh** [comment]
reviewed

**Review by hardik-desai-kcl — DISMISSED** [review]

**Comment by AkshayBanifatsyevichChugh** [comment]
reviewed

**Review by hardik-desai-kcl — DISMISSED** [review]

**Review by hardik-desai-kcl — APPROVED** [review]
